src/App.js

```diff
@@ -1,112 +1,57 @@
 import React from "react";
 import "./App.css";
-import Header from './components/Header';
-import Footer from './components/Footer';
-import ImageSlider from './components/ImageSlider';
-import ProductDetail from './components/ProductDetail';
-import SubHeader from './components/SubHeader';
-import TopNav from './components/TopNav';
+import Header from "./components/Header";
+import Footer from "./components/Footer";
+import ImageSlider from "./components/ImageSlider";
+import SubHeader from "./components/SubHeader";
+import TopNav from "./components/TopNav";
+import ProductDetail from "./components/ProductDetail";
 
 function App(props) {
   function renderProducts(products){
     var _products = products.map((product, index) => {
       return  <div key={index}> <ProductDetail product={product} /> </div>
     })
 
-	return_products;
+    return _products;
+  }
 
+    return (
       <div className="App">
         	<div className="wrap">
-	
-			<div className="header">
-			
-			<Header />
-
+				<div className= "header">
+            <Header />
 			<div className="clear"> </div>
-			
 			<SubHeader />
-
 			<div className="clear"> </div>
-			
-			<TopNav  />
-		
+			<TopNav changeCategory={props.changeCategory} />
```

Comment:
There is a problem here. Do you see what it is?
Why did you type props.**changeCategory**   ?
Where did changeCategory come from ?